### PR TITLE
Familienentlastungsgesetz

### PR DESCRIPTION
**Pull request by Eric-Sommer**
addresses #124. Implements Art. 3 [FamEntlastG](https://www.buzer.de/3_Familienentlastungsgesetz.htm) becoming effective 1 January 2020.

Also some cleanup of `e_st.yaml` and `e_st_abzuege.yaml`.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -20,6 +20,8 @@ releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_
 * Child tax allowance modelled as two separate items. (:ghuser:`Eric-Sommer`)
 * Alimony advance payment (*Unterhaltsvorschuss*) now modelled more in line
   with the law (:ghuser:`Eric-Sommer`)
+* Implement *Familienentlastungsgesetz* on income tax tariff and child tax
```
Comment (mjbloemer) [inline CHANGES.rst]: This is only Artikel 3 of FamEntlastG, right? Maybe just say which functions are affected.

File: gettsim/data/e_st_abzuege.yaml [hunk 2]
```diff
@@ -553,21 +585,21 @@ hhfreib:
       note:
     2015-01-01:
       value: 1908
-      note: geändert durch BGBl. Nr. 30, 16.07.2015
+      note: Art. 1 G. v. 16.07.2015 BGBl. I S.1202
 hhfreib_ch:
   name:
-    de: Entlastungsbetrag für Alleinerziehende, Zusatzbetrag pro Kind ab dem 2. Kind  §24b (2) S.2 EStG
-    en:
+    de: Entlastungsbetrag für Alleinerziehende, Zusatzbetrag pro Kind ab dem 2. Kind
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]: This is a bit too long for a name. If you touch this in this PR: Can you move parts of that to the description? You can also simply keep the old value/string and do the cleanup in a separate PR.

File: gettsim/data/e_st_abzuege.yaml [hunk 3]
```diff
@@ -553,21 +585,21 @@ hhfreib:
...
     en:
   values:
     2015-01-01:
       value: 240
-      note:
+      note: Art. 1 G. v. 16.07.2015 BGBl. I S.1202
 kuerzquo:
   name:
-    de: Pauschale Kuerzung des Vorwegabzugs um # Prozent der Einnahmen  SEB Vorsicht - falsch modelliert, bleibt weiterhin bei 0.16% Gilt für AN die vom AG steuerfreie Zukunftssicherungsleistungen bekommen (z.B. AG-Anteile) also für fast alle oder Leute die in der RV versicherungsfrei sind  (Höhe § 10 (3) Nr. 2 EStG)
+    de: Pauschale Kuerzung des Vorwegabzugs um
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]: um?

File: gettsim/data/e_st_abzuege.yaml [hunk 4]
```diff
@@ -668,21 +700,21 @@ vorsorg_rv_sh:
...
+    de: §10 Abs. 4 S.1 EStG
     en:
   values:
     2005-01-01:
       value: 1900
       note:
 rentert:
   name:
-    de: Ertragsanteil der Rente bei Rentenbeginn im 65. LJ.  ES Macht keinen Sinn, zu parametrisieren, weil es um das Renteneintrittsalter geht  (§ 22 Nr. 1 a) aa) EStG)
+    de: Ertragsanteil der Rente bei Rentenbeginn im 65. LJ. Macht keinen Sinn, zu parametrisieren, weil es um das Renteneintrittsalter geht.
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]: name too long. can also be in a separate cleanup PR.

File: gettsim/data/e_st_abzuege.yaml [hunk 5]
```diff
@@ -755,10 +787,10 @@ sauntmx:
       note: .
 sbhp100:
   name:
-    de: Pauschbeträge für behinderte Menschen, Hinterbliebene und Pflegepersonen (Grad der Behinderung 95% oder 100%)  (Höhe § 33b (1), (3) EStG)
+    de: Pauschbeträge für behinderte Menschen, Hinterbliebene und Pflegepersonen (Grad der Behinderung 95% oder 100%)
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]: name too long. can also be in a separate cleanup PR. also the following instances of sbhp...

File: gettsim/data/e_st_abzuege.yaml [hunk 6]
```diff
@@ -346,12 +354,24 @@ altenth:
     2019-01-01:
       value: 836
       note:
+    2020-01-01:
+      value: 760
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

If you add a paramter/date/value, please also fill the note.

If 24a EStG S. 5 adds all the parameters for a long period, always cite the BGBl that defines these parameters.

also following instances.

File: gettsim/data/e_st_abzuege.yaml [hunk 7]
```diff
@@ -399,9 +419,21 @@ altentq:
     2019-01-01:
       value: 0.176
       note:
+    2020-01-01:
+      value: 0.16
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 8]
```diff
@@ -399,9 +419,21 @@ altentq:
...
+    2021-01-01:
+      value: 0.152
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 9]
```diff
@@ -399,9 +419,21 @@ altentq:
...
+    2022-01-01:
+      value: 0.144
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 10]
```diff
@@ -399,9 +419,21 @@ altentq:
...
+    2023-01-01:
+      value: 0.136
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 11]
```diff
@@ -346,12 +354,24 @@ altenth:
     2019-01-01:
       value: 836
       note:
+    2020-01-01:
+      value: 760
+      note:
+    2021-01-01:
+      value: 722
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 12]
```diff
@@ -346,12 +354,24 @@ altenth:
...
+    2022-01-01:
+      value: 684
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 13]
```diff
@@ -346,12 +354,24 @@ altenth:
...
+    2023-01-01:
+      value: 646
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 14]
```diff
@@ -327,7 +335,7 @@ altenth:
       note: Stand Neugefasst durch Bek. v. 8.10.2009 I 3366, 3862,   zuletzt geändert durch Art. 20 G v. 20.12.2011 I 2854
     2013-01-01:
       value: 1292
-      note: EStG §24a Satz 5 (Tabelle)
+      note:
```
Comment (mjbloemer) [inline gettsim/data/e_st_abzuege.yaml]:
```suggestion
      note: Artikel 1 G. v. 05.07.2004 BGBl. I S. 1427.
```

File: gettsim/data/e_st_abzuege.yaml [hunk 15]
```diff
@@ -123,6 +126,8 @@ vorwegab:
     2019-01-01:
       value: 300
       note:
+    2020-01-01:
+      value: 0
```
[the remark this answers is not in the document]
Comment (Eric-Sommer) [reply gettsim/data/e_st_abzuege.yaml]: There is no note because these are values from a table given in §10. And you are right, the last value is 300 and not zero. One could handle this via a year switch, but it would actually be something for #43.

File: gettsim/data/e_st_abzuege.yaml [hunk 16]
```diff
@@ -173,10 +178,13 @@ vwabzc:
     2019-01-01:
       value: 600
       note:
+    2020-01-01:
+      value: 0
+      note:
```
[the remark this answers is not in the document]
Comment (Eric-Sommer) [reply gettsim/data/e_st_abzuege.yaml]: see above

File: CHANGES.rst [hunk 17]
```diff
@@ -20,6 +20,8 @@ releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_
 * Child tax allowance modelled as two separate items. (:ghuser:`Eric-Sommer`)
 * Alimony advance payment (*Unterhaltsvorschuss*) now modelled more in line
   with the law (:ghuser:`Eric-Sommer`)
+* Implement Art. 3 of *Familienentlastungsgesetz* on income tax tariff and child tax
+  allowance becoming effective in 2020
```
Comment (mjbloemer) [inline CHANGES.rst]: and cleanup of e_st_abzuege parameters
